server/portal/settings/settings.py

```diff
@@ -24,6 +24,12 @@ def gettext(s): return s  # noqa:E731
 
 # Build paths inside the project like this: os.path.join(BASE_DIR, ...)
 BASE_DIR = os.path.dirname(os.path.dirname(os.path.abspath(__file__)))
+
+if os.path.isfile(os.path.join(BASE_DIR, 'settings', 'settings_custom')):
+    from portal.settings import settings_custom
+else:
+    from portal.settings import settings_default as settings_custom
+
```

Comment:
__Question__
Is it logged which settings was loaded (perhaps via [`logger`](https://github.com/TACC/Core-Portal/blob/task/FP-304--custom-settings/server/portal/settings/settings.py#L19))? If not, then is it worth the effort to log which settings was loaded? [CMS does so (albeit with Python code that could be improved).](https://github.com/TACC/Core-CMS/blob/main/taccsite_cms/settings.py#L23-L50)

Reply:
I want to keep these separate and explicit. `settings_custom` will differ greatly based on the portal using it, and it won't be used locally.

Reply:
we can add those logs, though I'm not sure in what case they'd be referenced

Reply:
No request for logs. I only used those specific CMS logs when initially testing custom secrets, and haven't looked at them since. I proposed it to see if others found a benefit beyond what I knew.

Reply:
The statement "`settings_custom` will differ greatly based on the portal using it" resolves my concern. _For the amount of settings here (now that I know they will differ greatly), I see a benefit to having them separate and explicit (easier to compare and keep organized consistently)._